src/core/a-entity.js

```diff
@@ -50,8 +49,14 @@ var proto = Object.create(ANode.prototype, {
     }
   },
 
+  /**
+   * Handle changes coming from the browser DOM inspector.
+   */
   attributeChangedCallback: {
     value: function (attr, oldVal, newVal) {
+      // TODO: Remove this return once we bump DRE polyfill. The DRE polyfill is currently
+      // calling our wrapped AEntity.getAttribute which sometimes returns a parsed object.
+      if (typeof newVal === 'object') { return; }
```

Comment:
The shaders PR has landed and we now use a fork of the registerElement polyfill that makes this change unnecessary